@clr/angular -  Compilation problem due to TS2611

## @clr/angular -  Compilation problem due to TS2611

After updating from Clarity 2.4.5 to 4.0.1 I get problem with building Angular application. I get

```
ERROR in node_modules/@clr/angular/accordion/stepper/stepper-panel.d.ts:18:9 - error TS2611: 'id' is defined as a property in class 'ClrAccordionPanel', but is overridden here in 'ClrStepperPanel' as an accessor.

18     get id(): string;
           ~~
```

## How to reproduce
Steps to reproduce the behavior:

1.  Update `@clr/angular`, `@clr/icons`, `@clr/ui` from `2.4.5` to `4.0.1`.
2.  Update `@webcomponents/custom-elements` from `1.0.0.` to `1.4.2`
3.  Run `ng build`
4.  See error

## Expected behavior

Everything goes smooth :)

## Versions

**App**

- Angular: 10.1.0
- Node: 12.18.3
- Clarity: 4.0.1

--- a/packages/angular/golden/clr-angular.d.ts
+++ b/packages/angular/golden/clr-angular.d.ts
@@ -89,12 +89,13 @@ export declare class ClrAccordionPanel implements OnInit, OnChanges {
     accordionDescription: QueryList<ClrAccordionDescription>;
     commonStrings: ClrCommonStringsService;
     disabled: boolean;
-    id: string;
+    get id(): string;
+    set id(value: string);
     isAccordion: boolean;
     panel: Observable<AccordionPanelModel>;
     panelOpen: boolean;
     panelOpenChange: EventEmitter<boolean>;
-    constructor(commonStrings: ClrCommonStringsService, accordionService: AccordionService, ifExpandService: IfExpandService, id: string);
+    constructor(commonStrings: ClrCommonStringsService, accordionService: AccordionService, ifExpandService: IfExpandService, _id: string);
     collapsePanelOnAnimationDone(panel: AccordionPanelModel): void;
     getAccordionContentId(id: string): string;
     getAccordionHeaderId(id: string): string;

--- a/packages/angular/projects/clr-angular/src/accordion/accordion-panel.ts
+++ b/packages/angular/projects/clr-angular/src/accordion/accordion-panel.ts
@@ -47,11 +47,19 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   readonly AccordionStatus = AccordionStatus;
   isAccordion = true;
 
+  get id(): string {
+    return this._id;
+  }
+
+  set id(value: string) {
+    this._id = value;
+  }
+
   constructor(
     public commonStrings: ClrCommonStringsService,
     private accordionService: AccordionService,
     private ifExpandService: IfExpandService,
-    @Inject(UNIQUE_ID) public id: string
+    @Inject(UNIQUE_ID) private _id: string
   ) {}
 
   ngOnInit() {

